### PR TITLE
virtual properties translations

### PR DESCRIPTION
**Pull request by mom666**
virtual properties for value translations

### CODE REVIEW

File: src/Entity/Value.php [hunk 1]
```diff
@@ -60,6 +60,15 @@
      */
     protected $attribute;
 
+    /**
+     * @Serializer\VirtualProperty
+     * @Serializer\SerializedName("attributeTranslations")
+     * @Serializer\Groups("translations_value_view")
```
Comment (tomfun) [inline src/Entity/Value.php]: translations_value_view -> eav_value_view_title_translations
eav_value - entity code
view - means extra groups
title_translations - return transltions of title

File: src/Entity/Value.php [hunk 2]
```diff
@@ -60,6 +60,15 @@
...
+     */
+    public function getAttributeTranslations()
+    {
+        return $this->getAttribute()->getTranslations();
```
Comment (tomfun) [inline src/Entity/Value.php]: we don't need all translations, just title for entity view page

File: src/Entity/ValueSelect.php [hunk 3]
```diff
@@ -20,6 +20,16 @@ class ValueSelect extends Value
     protected $option = null;
 
     /**
+     * @Serializer\VirtualProperty
+     * @Serializer\SerializedName("selectedOption")
+     * @Serializer\Groups("translations_value_view")
+     */
+    public function getSelectedOption()
+    {
+        return $this->getOption()->getTranslations();
```
Comment (tomfun) [inline src/Entity/ValueSelect.php]: we don't need all translations, just option title for entity view page

Comment (tomfun) [reply src/Entity/ValueSelect.php]: and please make manual testing of option creating

**Review by tomfun — CHANGES_REQUESTED** [review]

File: src/Entity/Value.php (hunk 2 above)
Comment (mom666) [reply src/Entity/Value.php]: But we need all locales. Am I need to parse them?

File: src/Entity/ValueSelect.php (hunk 3 above)
Comment (mom666) [reply src/Entity/ValueSelect.php]: What are you mean?

Comment (mom666) [reply src/Entity/ValueSelect.php]: I'm about testing

File: src/Entity/Value.php (hunk 2 above)
Comment (tomfun) [reply src/Entity/Value.php]: No, there _is method_ to get all locales as array. Use it. But if frontend need all fields from translation you must use all translations, this virtual property needed if only you want show **title** of attribute of value.

File: src/Entity/ValueSelect.php (hunk 3 above)
Comment (tomfun) [reply src/Entity/ValueSelect.php]: What will happen if we create options in manage panel?

**Review by tomfun — APPROVED** [review]

**Pull request merged**